clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -1628,13 +1615,13 @@ class AnnotatingParser {
       if (!Scopes.empty())
         Scopes.pop_back();
       // Lines can start with '}'.
-      if (Tok->Previous)
+      if (Prev)
```

Comment:
This actually fixes a bug. See https://github.com/llvm/llvm-project/pull/142104#issuecomment-2937232863.